pkg/kubelet/dockertools/manager.go

```diff
@@ -1202,7 +1202,9 @@ func (dm *DockerManager) killPodWithSyncResult(pod *api.Pod, runningPod kubecont
 	if networkContainer != nil {
 		ins, err := dm.client.InspectContainer(networkContainer.ID.ID)
 		if err != nil {
-			glog.Errorf("Error inspecting container %v: %v", networkContainer.ID.ID, err)
+			err = fmt.Errorf("Error inspecting container %v: %v", networkContainer.ID.ID, err)
+			glog.Error(err)
+			result.Fail(err)
```

Comment:
Yeah, it would be better if we could think of a less intrusive way to do this, or else people don't quite understand this may easily miss it.